spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/SourceToFunctionsSupportTests.java

```diff
@@ -234,7 +234,7 @@ public void testFiniteFluxSupplierSimple() {
 			assertThat(new String(target.receive(2000).getPayload())).isEqualTo("6");
 
 			assertThat(context.getBean("supplierInitializer")).isNotEqualTo(null);
-			assertThat(context.getBean("functionInitializer")).isEqualTo(null);
+//			assertThat(context.getBean("functionInitializer")).isEqualTo(null);
```

Comment:
What made you to comment this out, please?

Reply:
It is no longer true so I'll remove it all together